experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -43,6 +40,9 @@
 # libraryFilename(name)
 - library/${prefix}-${name}
 
+# libraryImport(name)
+- [${prefix}-${name}](${prefix}-${name})
```

Comment:
Do we need to consider relative path?

Reply:
Not here because this is importing a sibling.

---
In reply to: [478221085](https://github.com/microsoft/BotBuilder-Samples/pull/2690#discussion_r478221085) [](ancestors = 478221085)